content/docs/user-guide/data-management/remote-storage/amazon-s3.md

```diff
@@ -0,0 +1,261 @@
+```
+
+- `bucket` - name of an [existing S3 bucket]
+- `key` - optional path to a [folder key] in your bucket
+
+<admon type="tip">
+
+The `-d` flag (optional) makes this the `--default` remote for the
+<abbr>project</abbr>.
+
+</admon>
+
+[s3]: https://aws.amazon.com/s3/
+[existing s3 bucket]:
+  https://docs.aws.amazon.com/AmazonS3/latest/gsg/CreatingABucket.html
+[folder key]:
+  https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-folders.html
+
+## Authentication
+
+<admon type="info">
+
+The AWS user needs the following permissions: `s3:ListBucket`, `s3:GetObject`,
+`s3:PutObject`, `s3:DeleteObject`.
+
+</admon>
+
+DVC will try to authenticate using your [AWS CLI config] by default. This reads
+the default AWS credentials file (if available). To use customize
+authentication, you can set the following config params with
+`dvc remote modify --local`.
+
+[aws cli config]:
+  https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html
+
+<admon type="warn">
+
+The `--local` flag is needed to write sensitive user info to a Git-ignored
+config file (`.dvc/config.local`) so that no secrets are leaked through Git. See
+`dvc config` for more info.
+
+</admon>
+
+- `configpath` - custom [AWS CLI config file][aws-cli-config] path
+
+  ```cli
+  $ dvc remote modify --local myremote \
+                      configpath 'path/to/config'
+  ```
+
+- `credentialpath` - custom [AWS CLI credentials file][aws-cli-config] path
+
+  ```cli
+  $ dvc remote modify --local myremote \
+                      credentialpath 'path/to/credentials'
+  ```
+
+- `profile` - credentials [profile] name
+
+  ```cli
+  $ dvc remote modify --local myremote profile 'myprofile'
+  ```
+
+- `access_key_id` - AWS access key ID. May be used (along with
+  `secret_access_key`) instead of `credentialpath`.
+
+  ```cli
+  $ dvc remote modify --local myremote \
+                      access_key_id 'mysecret'
+  ```
+
+- `secret_access_key` - AWS secret access key. May be used (along with
+  `access_key_id`) instead of `credentialpath`.
+
+  ```cli
+  $ dvc remote modify --local myremote \
+                      secret_access_key 'mysecret'
+  ```
+
+- `session_token` - AWS [MFA] session token (when required). May be used (along
+  with `access_key_id` and `secret_access_key`) instead of `credentialpath`.
+
+  ```cli
+  $ dvc remote modify --local myremote \
+                      session_token 'mysecret'
+  ```
+
+[aws-cli-config]:
+  https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html
+[profile]:
+  https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html
+[mfa]: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_mfa.html
+
+## S3-compatible servers (non-Amazon)
+
+Set the `endpointurl` parameter with the URL to connect to the S3-compatible
+service (e.g. [MinIO], [IBM Cloud Object Storage], etc.). For example, let's set
+up a [DigitalOcean Space] (equivalent to a bucket in S3) called `mystore` found
```

Comment:
The Amazon S3 page includes the [non-Amazon, S3-compatible section](https://dvc-org-guide-data-mgmt-leaijc.herokuapp.com/doc/user-guide/data-management/remote-storage/amazon-s3#s3-compatible-servers-non-amazon), since it only involves one config option.